### PR TITLE
OpenMS 2.3

### PR DESCRIPTION
**Pull request by blankclemens**
A first shot at the wrappers for OpenMS 2.3.

Quite likely some tools have changed or won't work at all, so I once again ask for as much feedback as possible, since personally I'm not an active user of this suite at the moment.

I included all manual fixes that I'm aware of (I try to collect them in the readme).

### CODE REVIEW

File: tools/openms/LowMemPeakPickerHiResRandomAccess.xml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<!--This is a configuration file for the integration of a tools into Galaxy (https://galaxyproject.org/). This file was automatically generated using CTD2Galaxy.-->
+<!--This is a configuration file for the integration of a tools into Galaxy (https://galaxyproject.org/). This file was automatically generated using CTDConverter.-->
 <!--Proposed Tool Section: [Utilities]-->
-<tool id="LowMemPeakPickerHiRes_RandomAccess" name="LowMemPeakPickerHiRes_RandomAccess" version="2.2.0">
+<tool id="LowMemPeakPickerHiResRandomAccess" name="LowMemPeakPickerHiResRandomAccess" version="2.3.0">
```
Comment (bgruening) [inline tools/openms/LowMemPeakPickerHiResRandomAccess.xml]: mh, I guess we want to keep the ID right? I will change it.

File: tools/openms/MapAlignerPoseClustering.xml [hunk 2]
```diff
@@ -1,19 +1,43 @@
-<tool id="MapAlignerPoseClustering" name="MapAlignerPoseClustering" version="2.2.0">
+<?xml version='1.0' encoding='UTF-8'?>
+<!--This is a configuration file for the integration of a tools into Galaxy (https://galaxyproject.org/). This file was automatically generated using CTDConverter.-->
+<!--Proposed Tool Section: [Map Alignment]-->
+<tool id="MapAlignerPoseClustering" name="MapAlignerPoseClustering" version="2.3.0">
```
Comment (bgruening) [inline tools/openms/MapAlignerPoseClustering.xml]: Great work @blankclemens! This is the only tool which I think might make problems.
Following the documentation I think this tool can take N files and will create N files and not one. http://ftp.mi.fu-berlin.de/pub/OpenMS/release-documentation/html/TOPP_MapAlignerPoseClustering.html

I made these changes here for the old version: https://github.com/galaxyproteomics/tools-galaxyp/pull/254/files
If this makes sense to you, lets merge the older one and then rebase this one.

**Comment by bgruening** [comment]
ok, from my side this looks good :)

**Comment by bgruening** [comment]
Thanks a lot @blankclemens and @TorHou for reviewing!

**Comment by blankclemens** [comment]
Thanks for fixing and merging! This time I included the tools that where on the blacklist for having multiple outputs, maybe it’s worth checking them. I’m on the road for the next days though, hope the majority work as expected.

**Pull request merged**